fastlane/download_metadata.swift

```diff
@@ -105,22 +105,20 @@ func downloadTranslation(languageCode: String, folderName: String) {
         let fileManager = FileManager.default
         try? fileManager.createDirectory(atPath: languageFolder, withIntermediateDirectories: true, attributes: nil)
 
-        /// This sort of hack-ey – we're not actually downloading anything here, but the idea is
```

Comment:
This is removed since we've downloaded it from ASC and it won't be deleted anymore now.